python/tank/bootstrap/manager.py

```diff
@@ -583,11 +585,41 @@ def get_pipeline_configurations(self, project):
             current_login=self._sg_user.login,
             sg_connection=self._sg_connection,
         ):
+            try:
+                path = ShotgunPath.from_shotgun_dict(pc)
+                config_path = path.current_os
```

Comment:
I think you should split the `try` in two blocks. If there's a ValueError thrown from `from_shotgun_dict` you'll have a false positive and hide a bug.